examples/dbscan/dbscan.cpp

```diff
@@ -320,6 +323,7 @@ int main(int argc, char *argv[])
   // clang-format off
   desc.add_options()
       ( "help", "help message" )
+      ( "algorithm", bpo::value<std::string>(&algorithm)->default_value("hdbscan"), "algorithm (dbscan | hdbscan)" )
```

Comment:
This is not strictly true. I think just now we avoided implementing `minpts = 2` because we are not sure we need self-loops when doing flat clustering. We'll figure out a way to make it work.

Reply:
My point is the default arguments lead to a failed assertion